src/backbone.radio.js

```diff
@@ -168,7 +168,12 @@ _.extend(Radio, {
  */
 
 function makeCallback(callback) {
-  return _.isFunction(callback) ? callback : function () { return callback; };
+  if (_.isFunction(callback)) {
+    return callback;
+  }
+  var result = function () { return callback; };
+  result._callback = callback;
+  return result;
 }
```

Comment:
Actually looking at this quickly it looks like our `replyOnce` [implementation is incorrect](https://github.com/marionettejs/backbone.radio/blob/master/src/backbone.radio.js#L231)

See [backbone](https://github.com/jashkenas/backbone/blob/master/backbone.js#L330)

cc @jmeas @jridgewell.